packages/bits/demo/src/components/demo/wizard-v2/wizard-with-custom-icons/wizard-with-custom-icons-example.component.html

```diff
@@ -0,0 +1,40 @@
+<nui-wizard-horizontal [customIcons]="{selected: 'edit'}">
+
+    <nui-wizard-step-v2 i18n-label label="First step">
+        <span i18n>This is the first step of our wizard</span>.
+        <ng-template wizardStepFooter>
+            <ng-container *ngTemplateOutlet="wizardNext"></ng-container>
+        </ng-template>
+    </nui-wizard-step-v2>
+
+    <nui-wizard-step-v2 i18n-label label="Second step" [icon]="'email'">
```

Comment:
maybe `defaultIcon` would be better since it's only shown when the step isn't active or complete?

Reply:
I think we'll take the next sprint to think the api through a bit better. This input, no matter how we call it, immediately makes me think of why are we allowing to change exactly this state out of 4 possible states. During our call we assumed this may be the only state consumers would want to change for the step, though, just like every assumption, this one feels a bit shaky, because our guess can be wrong, and then later we'll need to expand, or (the worst case) change the api which is not desirable.

Reply:
I like your thinking on this. ❤️

Reply:
Another thing to think about is whether we really need to have this little piece that's customizable if it's already possible to customize the entire wizard (see custom layout example). As you mentioned in the refinements meeting we don't really have any data on who will want to customize what. As a resultl, maybe we could even postpone this feature until we know what consumers will want to do?